src/main/java/com/iridium/iridiumskyblock/configs/Missions.java

```diff
@@ -18,6 +18,8 @@
  */
 public class Missions {
 
+    public List<String> blockedWorlds = Collections.emptyList();
```

Comment:
We already have three IridiumSkyblock worlds, what's the point in this additional option?

Reply:
like kill mission, when player kill an entity on spawn or anywhere it increments the mission
someone wanted blocking it

Reply:
Yes, but you should rather check for one of the three world names that already exist instead of introducing a new one

Reply:
I thought it too, but may users wants not blocking some worlds

Reply:
Why would we want to make it a lot harder and confusing for most people if maybe one person wants that feature?

Just imagine you're renaming the worlds in the configuration.yml and now you need to know that the names in the missions.yml also have to be updated...

Reply:
umm someone wants it i didn't think this side, you're right about this i'll pull back :panda:

Reply:
or if you have another thing?

Reply:
Another thing? I probably don't

Reply:
Just use the three configured worlds from the configuration.yml